scripts/core/editor3/components/embeds/EmbedInput.tsx

```diff
@@ -131,6 +132,14 @@ export class EmbedInputComponent extends React.Component<any, any> {
 
         return (
             <form onSubmit={this.onSubmit} className="embed-dialog" onKeyUp={this.onKeyUp}>
+                <OverlayTrigger overlay={
+                    <Tooltip id="create_new_embed_tooltip">
+                        <p translate>To get a responsive embed code, paste a URL.</p>
+                        <p translate>If you paste an embed code, it will be used &quot;as is&quot;.</p>
```

Comment:
`translate` attribute only works in angular templates. Use `gettext` elsewhere. `<p>{gettext('To get a responsive embed code, paste a URL.')}</p>`